src/Symfony/Component/Form/AbstractType.php

```diff
@@ -64,8 +64,16 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * {@inheritdoc}
      */
+    public function getName()
+    {
+        return;
```

Comment:
Of course it does! We can never, ever remove a method from an interface. Consider:

```php
public function doSomething(FormTypeInterface $type)
{
    // boom
    $name = $type->getName();
}
```